support/cas-server-support-hazelcast/src/main/java/org/apereo/cas/config/HazelcastConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package org.apereo.cas.config;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import lombok.extern.slf4j.Slf4j;
+import org.apereo.cas.configuration.CasConfigurationProperties;
+import org.apereo.cas.configuration.model.support.hazelcast.BaseHazelcastProperties;
+import org.apereo.cas.hz.HazelcastConfigurationFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Creates a singular Hazelcast Instance that other hazelcast modules add maps to
+ * instead of creating thier own instance.
+ *
+ * @author Travis Schmidt
+ * @since 5.3.4
+ */
+@Configuration("hazelcastConfiguration")
+@EnableConfigurationProperties(CasConfigurationProperties.class)
+@Slf4j
+public class HazelcastConfiguration {
+
+    @Autowired
+    private CasConfigurationProperties casProperties;
+
+    @Bean
+    public HazelcastInstance hazelcastInstance() {
```

Comment:
Bean name here too is kind of risky. Was that always there? If not, we probably should rename this as well.